### ISSUE TITLE
Update darktable to 2.6.1

### ISSUE THREAD

**Issue by Sangeppato** [lukesampson/scoop-extras#1831]
Darktable 2.6.1 was released: https://www.darktable.org/2019/03/darktable-261-released/

**Comment by Ash258** [comment]
They moved from zip to exe. Here is fixed manifest, but there is needed to determine how to install. I don't have time right now to test.

```json
{
    "homepage": "https://www.darktable.org/",
    "description": "Open source photography workflow application and raw developer. A virtual lighttable and darkroom for photographers",
    "license": "GPL-3.0-only",
    "version": "2.6.1",
    "url": "https://github.com/darktable-org/darktable/releases/download/release-2.6.1/darktable-2.6.1-win64.exe",
    "hash": "c0b31a3ac4f02151f4fc0ee4e8e39dc1e5c8ef409b2f23eee440affba9cd73f6",
    "bin": "bin\\darktable.exe",
    "shortcuts": [
        [
            "bin\\darktable.exe",
            "darktable"
        ]
    ],
    "checkver": {
        "github": "https://github.com/darktable-org/darktable",
        "regex": "tree/release-([\\d\\.]+)"
    },
    "autoupdate": {
        "url": "https://github.com/darktable-org/darktable/releases/download/release-$version/darktable-$version-win64.exe",
        "hash": {
            "url": "https://github.com/darktable-org/darktable/releases/tag/release-$version"
        }
    }
}
```

**Comment by r15ch13** [comment]
Found something about NSIS 3.04 https://sourceforge.net/p/sevenzip/discussion/45797/thread/3f97de5252/
But I don't know what the `NSIS signature` is. Yet :smile:

**Comment by rpeyron** [comment]
Found a way to install with the new exe. Here is the corresponding PR : https://github.com/lukesampson/scoop-extras/pull/4707

**Comment by flavio20002** [comment]
This can be closed I guess, Was resolved with #4707
